### PR TITLE
Make GPG Pass store path configurable by git config

### PR DESCRIPTION
**Pull request by ipitsyn**
I found this change beneficial when using conditional git configuration includes for multiple git identities and `pass` password stores. It lets me transparently select appropriate password store without need to set its path explicitly by environment variable.
For example:

> ~/.config/git/config:
```
[user]
	name = John Doe
	email = jdoe@gmail.com

[credential]
	credentialStore = gpg
	helper = /usr/local/bin/git-credential-manager
	gpgPassStorePath = /home/jdoe/.password-store

[includeIf "gitdir:~/Work/"]
	path = config.work
```

> ~/.config/git/config.work:
```
[user]
	name = John Doe (Umbrella Corp.)
	email = jdoe@umbrella.com

[credential]
	gpgPassStorePath = /home/jdoe/.password-store.umbrella
```

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.5.1.0
```
Comment (mjcheetham) [inline VERSION:1]: Please revert this change to the `VERSION` file.. this is only updated by maintainers just before a release.

**Review by mjcheetham — CHANGES_REQUESTED** [review]
This looks great thanks! Please can you undo the change to the `VERSION` file, and also can I ask that you add a new section to the `docs/config.md` file for the new config setting entry?

File: VERSION (hunk 1 above)
Comment (tduckworth2) [reply VERSION:1]: I need you to call me asap this is Blackstone CYM my number is 2798677377 I got a M dollar prop. For you

Comment (Comanche85) [reply VERSION:1]: Tried to call whats the M dollar prop

Comment (tduckworth2) [reply VERSION:1]: 5 mill the money I have a proposition for that much in dollars

**Comment by Comanche85** [comment]
Ok send me a $5000 retainer right now so I know you are serious

**Review by mjcheetham — APPROVED** [review]

**Comment by mdalaminak9089674532** [comment]
my business account, made the 

> [](url)Amin

> [``](

> _ #url_#32991933

)
